test/core/test_update.py

```diff
@@ -126,6 +138,188 @@ def test_update_global_attributes(self):
+        x_bnds = np.array([[v - res05, v + res05] for v in x])
+        y_bnds = np.array([[v - res05, v + res05] for v in y])
+        time = [pd.to_datetime(f'2018-06-0{i}T12:00:00') for i in
+                range(1, num_times + 1)]
+        time_bnds = [(pd.to_datetime(f'2018-06-0{i}T00:00:00'),
+                      pd.to_datetime(f'2018-06-0{i}T23:00:59')) for i in
+                     range(1, num_times + 1)]
+
+        coords = dict(time=(['time'], time),
+                      y=(['y'], y),
+                      x=(['x'], x))
+
+        coords_with_bnds = dict(time_bnds=(['time', 'bnds'], time_bnds),
+                                y_bnds=(['y', 'bnds'], y_bnds),
+                                x_bnds=(['x', 'bnds'], x_bnds),
+                                **coords)
+
+        output_metadata = dict(history='pipo', license='MIT',
+                               Conventions='CF-1.7')
+
+        expected_crs = "CRS84"
+
+        expected_bbox = (
+            'POLYGON((-121.99380296455976 -62.293519314442854, -121.99380296455976 '
+            '-62.299510171081884, -121.99083040333379 -62.299510171081884, '
+            '-121.99083040333379 -62.293519314442854, -121.99380296455976 '
+            '-62.293519314442854))')
+
+        ds1 = xr.Dataset(coords=coords)
+        ds1.rio.write_crs("epsg:3031", inplace=True,
+                          grid_mapping_name="crs").reset_coords()
+        ds2 = update_dataset_attrs(ds1, global_attrs=output_metadata)
+
+        self.assertIsNot(ds2, ds1)
+        self.assertEqual('CF-1.7', ds2.attrs.get('Conventions'))
+        self.assertEqual('MIT', ds2.attrs.get('license'))
+        self.assertEqual('pipo', ds2.attrs.get('history'))
+        self.assertEqual(-121.99380296455976,
+                         ds2.attrs.get('geospatial_lon_min'))
+        self.assertEqual(-121.99083040333379,
+                         ds2.attrs.get('geospatial_lon_max'))
+        self.assertEqual(0.0005945389425789926,
+                         ds2.attrs.get('geospatial_lon_resolution'))
+        self.assertEqual('degrees_east', ds2.attrs.get('geospatial_lon_units'))
+        self.assertEqual(-62.293519314442854,
+                         ds2.attrs.get('geospatial_lat_min'))
+        self.assertEqual(-62.299510171081884,
+                         ds2.attrs.get('geospatial_lat_max'))
+        self.assertEqual(0.0011981728729608676,
+                         ds2.attrs.get('geospatial_lat_resolution'))
+        self.assertEqual(expected_crs, ds2.attrs.get('geospatial_bounds_crs'))
+        self.assertEqual(expected_bbox, ds2.attrs.get('geospatial_bounds'))
+        self.assertEqual('degrees_north', ds2.attrs.get('geospatial_lat_units'))
+        self.assertEqual('2018-06-01T00:00:00.000000000',
+                         ds2.attrs.get('time_coverage_start'))
+        self.assertEqual('2018-06-06T00:00:00.000000000',
+                         ds2.attrs.get('time_coverage_end'))
```

Comment:
Would be more readable if you compare `ds2.attrs` (minus the `'date_modified'` entry) against the expected dict.